completions/_vagrant

```diff
@@ -14,6 +14,13 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+        	boxes=(
+			$(print -l $(basename ~/.vagrant.d/boxes/*(/)))
```

Comment:
For me, print -l was a lot faster than waiting for the vagrant command to respond? Dan mentioned this as well, are you guys seeing vagrant box list being faster?